src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -230,16 +230,23 @@ default String getEffectiveAvatarUrl()
      * Represents the bit offsets used by Discord for public flags
      */
     enum Flags{
-        NONE(            0, "None"),
-        EMPLOYEE(        1, "Discord Employee"),
-        PARTNER(         2, "Discord Partner"),
-        HYPESQUAD(       4, "HypeSquad Events"),
-        BUG_HUNTER_1(    8, "Bug Hunter Level 1"),
-        BRAVERY(        64, "HypeSquad House Bravery"),
-        BRILLIANCE(    128, "HypeSquad House Brilliance"),
-        BALANCE(       256, "HypeSquad House Balance"),
-        SYSTEM(       4096, "System"),
-        BUG_HUNTER_2(16384, "Bug Hunter Level 2");
+        NONE(                      0, "None"),
+        STAFF(                     1, "Discord Employee"),
+        PARTNER(                   2, "Discord Partner"),
+        HYPESQUAD(                 4, "HypeSquad Events"),
+        BUG_HUNTER_LEVEL_1(        8, "Bug Hunter Level 1"),
+        // Bravery
+        HYPESQUAD_ONLINE_HOUSE_1( 64, "HypeSquad Online House Bravery"),
+        // Brilliance
+        HYPESQUAD_ONLINE_HOUSE_2(128, "HypeSquad Online House Brilliance"),
+        // Balance
+        HYPESQUAD_ONLINE_HOUSE_3(256, "HypeSquad Online House Balance"),
```

Comment:
Let's call them `HYPESQUAD_BRAVERY` and such.

Reply:
Was renamed.
https://github.com/DV8FromTheWorld/JDA/pull/3